lib/OrdinaryDiffEqDifferentiation/src/derivative_wrappers.jl

```diff
@@ -268,12 +268,12 @@ function build_jac_config(alg, f::F1, uf::F2, du1, uprev,
                 autodiff_alg_reverse = dir_reverse
             end
 
-            jac_config_forward = DI.prepare_jacobian(uf, du1, autodiff_alg_forward, u)
-            jac_config_reverse = DI.prepare_jacobian(uf, du1, autodiff_alg_reverse, u)
+            jac_config_forward = DI.prepare_jacobian(uf, du1, autodiff_alg_forward, u, strict=Val(false))
```

Comment:
is the `strict` argument new in DI 0.7? If so, this won't work with DI 0.6, right?